src/Dfe.PlanTech.AzureFunctions/Functions/QueueReceiver.cs

```diff
@@ -80,16 +97,63 @@ private async Task ProcessMessage(ServiceBusReceivedMessage message, ServiceBusM
         }
         catch (Exception ex)
         {
-            Logger.LogError(ex, "Error processing message ID {Message}", message.MessageId);
-            await messageActions.DeadLetterMessageAsync(message, null, ex.Message, ex.StackTrace, cancellationToken);
+            if (ex is JsonException or CmsEventException)
```

Comment:
Could this be better done by specifically catching the exceptions?

Something like

```
catch (Exception ex) when (ex is JsonException || ex is CmsEventException)
{
    ...
}
catch(Exception ex)
{
    ...
}
```

or 

```
catch(JsonException)
{
    ...
}
catch(CmsEventException)
{
    ...
}
catch(Exception)
{
    ...
}
```